velox/exec/HashProbe.cpp

```diff
@@ -1881,82 +1881,7 @@ void HashProbe::maybeSetupSpillOutputReader() {
   spillOutputPartitionSet_.clear();
 }
 
-SpillPartitionSet HashProbe::spillTable() {
```

Comment:
I think it is better to keep spill hash table control logic inside hash probe and only the hash table spill itself could be shared among build, probe and bridge